wolox-books/src/app/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const SPANISH_FORM_ERROR_MESSAGES = {
+    EMAIL: 'No cumple el formato de email (ejemplo@dominio.co)',
+    PASSWORD: 'Es necesario minimo un número, una minuscúla y una mayuscúla',
+    PASSWORD_MUST_MATCH: 'Ambas password deben coincidir'
+};
+
+export const PATTERNS = {
```

Comment:
Each validation will have an error associated with it,maybe we could have an object array, each object containing a regex and an error.